lib/fileutils.rb

```diff
@@ -799,12 +805,12 @@ def link_entry(src, dest, dereference_root = false, remove_destination = false)
   #     cp src1.txt dest1
   #     cp src2.txt src2.dat dest2
   #
-  # Raises an exception if +src+ is a directory.
-  #
   # Related: FileUtils.cp_r (recursive).
   #
   # FileUtils.copy is an alias for FileUtils.cp.
   #
+  # Raises an exception if +src+ is a directory.
```

Comment:
I prefer this line at the original location rather than here. I feel like this is more important that related methods and aliases.

Reply:
I mis-remembered the doc guidance, which actually says exceptions/aliases/relateds.